src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -180,4 +182,33 @@ public function handlePayoutPaid(Event $event): Response
             return $this->respondWithData($event->data->object);
         }
     }
+
+    public function handleChargeRefunded(Event $event): Response
+    {
+        $chargeId = $event->data->object->id;
+
+        /** @var Donation $donation */
+        $donation = $this->donationRepository->findOneBy(['chargeId' => $chargeId]);
+
+        if (!$donation) {
+            $this->logger->info(sprintf('Donation not found with Charge ID %s', $chargeId));
+            return $this->respond(new ActionPayload(204));
+        }
+
+        // For now we support the happy success path,
+        // as this is the only event type we're handling right now,
+        // convert status to the one SF uses.
+        if ($event->data->object->status === 'succeeded') {
```

Comment:
Wrong status? If not then we need an update comment to explain how this relates to the Stripe statuses for the charge & refund.